scripts/ci/service_name.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Check format of service_name.json. Command and AzureServiceName are required. Others are optional.
+Each highest level command group should have reference in service_name.json.
+"""
+import json
+
+from azure.cli.core import MainCommandsLoader, AzCli
+from azure.cli.core._help import AzCliHelp
+from azure.cli.core.commands import AzCliCommandInvoker
+from azure.cli.core.file_util import create_invoker_and_load_cmds_and_args, get_all_help
+from azure.cli.core.parser import AzCliCommandParser
+from mock import patch
+
+
+def main():
+    az_cli = AzCli(cli_name='az',
+                   commands_loader_cls=MainCommandsLoader,
+                   invocation_cls=AzCliCommandInvoker,
+                   parser_cls=AzCliCommandParser,
+                   help_cls=AzCliHelp)
+    with patch('getpass.getuser', return_value='your_system_user_login_name'):
+        create_invoker_and_load_cmds_and_args(az_cli)
+    help_files = get_all_help(az_cli)
+    high_command_set = set()
```

Comment:
What is `high`?

Reply:
`high` means `left`. The first word after `az`.